packages/fx-core/src/plugins/resource/aad/utils/aadManifestHelper.ts

```diff
@@ -152,4 +156,104 @@ export class AadManifestHelper {
 
     return result;
   }
+
+  public static validateManifest(manifest: AADManifest): string {
+    let warningMsg = "";
+
+    // if manifest doesn't contain name property or name is empty
+    if (!manifest.name || manifest.name === "") {
+      warningMsg += AadManifestErrorMessage.NameIsMissing;
+    }
+
+    // if manifest doesn't contain signInAudience property
+    if (!manifest.signInAudience) {
+      warningMsg += AadManifestErrorMessage.SignInAudienceIsMissing;
+    }
+
+    // if manifest doesn't contain requiredResourceAccess property
+    if (!manifest.requiredResourceAccess) {
+      warningMsg += AadManifestErrorMessage.RequiredResourceAccessIsMissing;
+    }
+
+    // if manifest doesn't contain oauth2Permissions or oauth2Permissions length is 0
+    if (!manifest.oauth2Permissions || manifest.oauth2Permissions.length === 0) {
+      warningMsg += AadManifestErrorMessage.Oauth2PermissionsIsMissing;
+    }
+
+    // if manifest doesn't contain preAuthorizedApplications
+    if (!manifest.preAuthorizedApplications) {
```

Comment:
pre auth apps should also contains outlook, etc.?
![image](https://user-images.githubusercontent.com/63089166/158958381-5bb0f7e6-3c87-4a5b-94ef-1ef56e08ae0d.png)